docs/app/partials/layout-introduction.tmpl.html

```diff
@@ -0,0 +1,364 @@
+    API keyword.
+    <br/>e.g.
+  </p>
+
+  <p>
+    This notation results in, for example, the following table for the <code>layout</code> and <code>flex</code> APIs:
+  </p>
+
+  <table class="md-api-table">
+      <thead>
+      <tr>
+        <th>layout API</th>
+        <th>flex API</th>
+        <th>Activates when device</th>
+      </tr>
+      </thead>
+      <tr>
+        <td>layout</td>
+        <td>flex</td>
+        <td>Sets default layout direction &amp; flex unless overridden by another breakpoint.</td>
+      </tr>
+      <tr>
+        <td>layout-xs</td>
+        <td>flex-xs</td>
+        <td>width &lt; <b>600</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-xs</td>
+        <td>flex-gt-xs</td>
+        <td>width &gt;= <b>600</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-sm</td>
+        <td>flex-sm</td>
+        <td><b>600</b>px &lt;= width &lt; <b>960</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-sm</td>
+        <td>flex-gt-sm</td>
+        <td>width &gt;= <b>960</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-md</td>
+        <td>flex-md</td>
+        <td><b>960</b>px &lt;= width &lt; <b>1280</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-md</td>
+        <td>flex-gt-md</td>
+        <td>width &gt;= <b>1280</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-lg</td>
+        <td>flex-lg</td>
+        <td><b>1280</b>px &lt;= width &lt; <b>1920</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-lg</td>
+        <td>flex-gt-lg</td>
+        <td>width &gt;= <b>1920/b>px</td>
+      </tr>
+      <tr>
+        <td>layout-xl</td>
+        <td>flex-xl</td>
+        <td>width &gt;= <b>1920</b>px</td>
+      </tr>
+    </table>
+
+  <p>Below is an example usage of the Responsive Layout API:</p>
+
+  <hljs lang="html">
+    <div layout='column' class="zero">
+
+      <div flex="33" flex-md="{{ vm.box1Width }}" class="one"></div>
+      <div flex="33" layout="{{ vm.direction }}" layout-md="row" class="two">
+
+        <div flex="22" flex-md="10" hide-lg class="two_one"></div>
+        <div flex="30px" show hide-md="{{ vm.hideBox }}" flex-md="25" class="two_two"></div>
+        <div flex="20" flex-md="65" class="two_three"></div>
+
+      </div>
+      <div flex class="three"></div>
+
+    </div>
+  </hljs>
+
+  <br/>
+
+  <p>
+  This Layout API means that developers no longer have to worry about the details of FlexBox styling.
+  The developer uses the Layout HTML API to specify <b><i>intention</i></b> and the Layout engine handles all the issues of CSS flexbox styling.
+  </p>
+
+  <p class="layout_note">
+    The Layout engine will even report console warnings for Layout conflicts and issues that are not automatically resolved.
+  </p>
+
+
+  <br/><br/>
+  <hr>
+  <br/>
+
+  <h3>Under-the-Hood</h3>
+
+  <p>
+    Due to performance problems when using Attribute Selectors with <b>IE10</b> and <b>IE11</b> browsers, our attributes are
```

Comment:
Just say "Internet Explorer" instead of IE10 and IE11. It would be good if we could provide a link with more information about said performance problems.